### DISCUSSION TITLE
Any support/plans for support of mobile touchscreens?

### DISCUSSION THREAD

**Discussion by NovaAndrom3da** [hyprwm/Hyprland#1170] [General]
I'm looking to implement global iOS-like gestures with hyprland. Is this already possible, or is it planned? Thanks :)

**Comment by vaxerski** [comment]
it's planned

**Reply by NovaAndrom3da** [reply]
Okay, thanks. I was hoping it would be as from what I've seen so far, hyprland will be perfect for what I'm doing.

**Reply by vaxerski** [reply]
I also sometimes miss it on me lappy but since it's not a yoga I rarely remember lol

Also there is a kjhillion things that are planned, so like, cant have everything at once. I am atm pretty busy with working on `xdg-desktop-portal-hyprland` to bring window sharing to hyprland.

**Reply by NovaAndrom3da** [reply]
Ah, okay. Maybe I could help out some with this? Although I'm not well versed in {C, C++, C#} or wayland or anything, and I have finals coming up soon, so that's a maybe 😆

**Reply by vaxerski** [reply]
no need. Always welcome, but prioritize your personal life. Don't be like me

**Reply by ghost** [reply]
@vaxerski what's wrong with xdg-desktop-portal-wlr?

**Reply by vaxerski** [reply]
read the readme here https://github.com/hyprwm/hyprland-protocols

**Comment by skrekhere** [comment]
working on this :) should have something to show very soon
